kbn_pm/src/commands/reset_command.mjs

```diff
@@ -12,22 +12,20 @@ import Path from 'path';
 import { REPO_ROOT } from '../lib/paths.mjs';
 import { dedent } from '../lib/indent.mjs';
 import { cleanPaths } from '../lib/clean.mjs';
-import * as Bazel from '../lib/bazel.mjs';
 import { findPluginCleanPaths, readCleanPatterns } from '../lib/find_clean_paths.mjs';
 
 /** @type {import('../lib/command').Command} */
 export const command = {
   name: 'reset',
-  description:
-    'Deletes node_modules and output directories, resets internal and disk caches, and stops Bazel server',
+  description: 'Deletes node_modules and output directories, resets internal and disk caches',
   reportTimings: {
     group: 'scripts/kbn reset',
     id: 'total',
   },
   flagsHelp: `
     --quiet              Prevent logging more than basic success/error messages
   `,
-  async run({ args, log }) {
+  async run({ log }) {
```

Comment:
I believe we should left for sometime a safe delete instruction on reset command for the following folders

```
bazel-bin
bazel-kibana
bazel-out
bazel-testlogs
```

There might be others I could be missing